packages/ui/react/.storybook/README.md

```diff
@@ -27,3 +27,5 @@ Storybook configuration files
 | main.ts, manager.ts, preview.ts | https://storybook.js.org/docs/configure                                |
 | devopness.ts                    | https://storybook.js.org/docs/configure/user-interface/theming         |
 | preview-head.html               | https://storybook.js.org/docs/configure/story-rendering#adding-to-head |
+
+> Small doc contribution by Wallan — May 2025.
```

Comment:
@WallanDavid unfortunately we cannot accept a contribution that's not adding value to our `README` file and to other users who use this repository.

Every contribution should solve a real problem and not only add random notes to the repository files. To find good first issues, please read and follow the instructions in our contribution guide https://github.com/devopness/devopness/blob/main/CONTRIBUTING.md